pkg/runner/cluster_k8s.go

```diff
@@ -277,14 +275,12 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 	var eg errgroup.Group
 
 	eg.Go(func() error {
-		err := c.watchRunPods(ctx, ow, input, result, &template)
-
-		erru := c.updateRunResult(&template, result)
-		if erru != nil {
-			ow.Errorw("could not update run result", "err", erru)
+		err := c.collectOutcomes(ctx, result, &template)
+		if err != nil {
+			ow.Errorw("could not start collecting outcomes", "err", err)
 		}
 
-		return err
+		return c.watchRunPods(ctx, ow, input, result, &template)
```

Comment:
@hacdias could you explain the change here, just so that we are on the same page? We no longer have `updateRunResult`, so I guess `collectOutcomes` collects results for individual test plan instances, while they are running, and not at the end (as previously), right?

Reply:
Yes, as they run.